# Define the goal
string arm_id # what arm are we looking at? wam1, wam2?
string ring_id # what is the ring we want to wait for?
---
# Define the result
uint32 ring_available
---
# Define feedback
float32 ring_distance
